<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterFilesDialog</class>
 <widget class="QDialog" name="ParameterFilesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parameterdateien</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Calibration data</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QListWidget" name="listWidget_input"/>
      </item>
      <item row="1" column="0">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="b_add_input">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="b_delete_input">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Iteration data</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QListWidget" name="listWidget_iteration"/>
      </item>
      <item row="1" column="0">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="b_add_iteration">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="b_delete_iteration">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Real data</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QListWidget" name="listWidget_result"/>
      </item>
      <item row="1" column="0">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="b_add_result">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="b_delete_result">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
